include/openenclave/edger8r/common.h

```diff
@@ -81,7 +81,7 @@ OE_INLINE oe_result_t oe_add_size(size_t* total, size_t size)
  * buffer. Make sure that the buffer has enough space.
  */
 #define OE_SET_IN_POINTER(argname, argsize, argtype)                       \
-    if (pargs_in->argname)                                                 \
+    if (pargs_in->argname) /* TODO: This check is now extraneous. */       \
```

Comment:
This header, though internal, is installed and is user visible when debugging.
 I am not comfortable with introducing a new TODO in the last week of v0.6 development.

Reply:
@anakrish This is literally just noting that we do `if(pargs_in->argname)` in the generated code now. There's no harm in doing it here, but it's superfluous. If you want I can just remove the TODOs until if/when we decide to remove the extra checks.